packages/veritone-react-common/src/components/DataTable/styles/modules/_muiTypography.scss

```diff
@@ -0,0 +1,90 @@
+// https://github.com/muicss/mui/blob/master/src/sass/mui/mixins/_typography.scss
+
+$mui-mui-text-styles: (
```

Comment:
we already have this file in `styles/modules/_muiTypography.scss` - let's avoid duplicating, instead modifying the existing file with the difference.

Reply:
fixed